cdap-ui/app/cdap/components/AppHeader/AboutPageModal/index.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Modal, ModalBody } from 'reactstrap';
+import T from 'i18n-react';
+import { getModeWithCloudProvider } from 'components/Header/ProductDropdown/helper';
+import Footer from 'components/Footer';
+import { Theme } from 'services/ThemeHelper';
+
+require('./AboutPageModal.scss');
+
+declare global {
+  /* tslint:disable:interface-name */
+  interface Window {
+    CDAP_CONFIG: any;
+  }
+}
+
+interface IAboutPageModalProps {
+  cdapVersion: string;
+  isOpen: boolean;
+  toggle: () => void;
+}
+
+const AboutPageModal: React.SFC<IAboutPageModalProps> = ({ cdapVersion, isOpen, toggle }) => {
+  const mode = getModeWithCloudProvider();
+  const productLogoSrc = Theme.productLogoAbout || '/cdap_assets/img/CDAP_darkgray.png';
+  return (
+    <Modal isOpen={isOpen} toggle={toggle} size="md" className="about-page-modal" backdrop="static">
+      <ModalBody>
+        <div className="close-section float-right" onClick={toggle}>
+          <span className="fa fa-2x fa-times" />
+        </div>
+        <div className="about-title">
+          <div className="cdap-logo-with-version">
+            <div className="logo-container">
+              <img src={productLogoSrc} />
+            </div>
+            <span className="cdap-version">
+              {T.translate('features.AboutPage.version', { version: cdapVersion })}
+            </span>
+          </div>
+        </div>
+        <div className="about-content">
+          <div className="cdap-mode-security">
+            <span className="cdap-mode">
+              <strong>{T.translate('features.AboutPage.mode')}</strong>
+              <span>{mode}</span>
+            </span>
+            <br />
+            <span className="cdap-security">
```

Comment:
This is a copy paste from old code. That being said yes we should totally avoid product names in code. There are still a lot of places where we reference cdap. This will require its own PR. Will resolve in a separate PR